### PR TITLE
Add scripts to upload and delete mock data for DORA dashboard

### PR DESCRIPTION
**Pull request by chalenge**
Add scripts to upload and delete mock data for DORA dashboard.

Ensure you are running the latest version of Faros CE.

From root directory
```
docker-compose stop; ./start.sh
```

Then get into the mock-data directory and install dependencies
```shell
cd mock-data && npm i
```

#### Upload data 
```shell
./bin/mock-data upload
```

#### Deleting data uploaded by script
```shell
./bin/mock-data delete
```

Use `-u` flag to specify Hasura service url if using a different endpoint

### CODE REVIEW

File: init/resources/hasura/endpoints/cicd_artifact.gql [hunk 1]
```diff
@@ -3,6 +3,7 @@ mutation insert_cicd_Artifact(
   $data_artifact_id: String!
   $data_artifact_organization: String!
   $data_artifact_source: String!
+  $data_origin: String!
```
Comment (chalenge) [inline init/resources/hasura/endpoints/cicd_artifact.gql:6]: @ypc-faros Does adding the origin as a mandatory field break the CLI scripts?

**Comment by tovbinm** [comment]
Let's move the the code from `init` to `mock`?

File: init/resources/hasura/endpoints/cicd_artifact.gql (hunk 1 above)
Comment (ypc-faros) [reply init/resources/hasura/endpoints/cicd_artifact.gql:6]: Yes, this will be a breaking change for events CLI. Maybe just make optional?

**Comment by chalenge** [comment]
> Let's move the the code from `init` to `mock`?

Ummm, fine by me but as `mock-data` until have multiple "mocking"  utilities. @cjwooo thoughts?

**Comment by tovbinm** [comment]
Sure, just not in `init`.

**Review by thomas-gerber — COMMENTED** [review]
I see a linter, compiler option and packaging info in `mock-data`. Do we need them here as well given we probably have similar ones in `init`?

**Comment by tovbinm** [comment]
@thomas-gerber you typically have these defined separately per project

File: mock-data/src/hasura.ts [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+  }
+
+  async deleteArtifactDeployment(origin: string): Promise<void> {
+    await this.api.post('delete_cicd_artifact_deployment', {
+      data_origin: origin,
+    });
+  }
+
+  async deleteCICDOrganization(origin: string): Promise<void> {
+    await this.api.post('delete_cicd_Organization', {
```
Comment (ypc-faros) [inline mock-data/src/hasura.ts:196]: Needs rename after the change in https://github.com/faros-ai/faros-community-edition/pull/54
delete_cicd_Organization => delete_cicd_organization

File: mock-data/src/hasura.ts [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+      data_origin: origin,
+    });
+  }
+
+  async deleteCICDRepository(origin: string): Promise<void> {
+    await this.api.post('delete_cicd_Repository', {
```
Comment (ypc-faros) [inline mock-data/src/hasura.ts:202]: delete_cicd_Repository => delete_cicd_repository

File: mock-data/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import {pino} from 'pino';
+import VError from 'verror';
+
+import {MockData} from './mockdata';
+
+const USAGE = 'Usage: node index.js <upload> | <delete> <hasura url> ';
```
Comment (ypc-faros) [inline mock-data/src/index.ts:6]: nit: <upload> | <delete> <hasura url> => <upload | delete> <hasura url>

**Review by ypc-faros — APPROVED** [review]

**Pull request merged**